blocks/ads-block/features/ads/ad-helper.js

```diff
@@ -87,6 +87,7 @@ export const formatSectionPath = (sectionPath) => {
     if (fmtPath.charAt(0) !== '/') {
       fmtPath = `/${fmtPath}`;
     }
+    fmtPath = fmtPath.replace(/\//, '');
```

Comment:
This all seems to work functionally but I think the `if (fmtPath.charAt(0) !== '/')` line seems to add a forward slash when it doesn't exist and then this line that you added would then remove that forward slash. I _think_ we may want to remove the `if (fmtPath.charAt(0) !== '/')` block unless there's a reason that you're aware of that it should remain.

Reply:
there should only be a `/` after websiteAdPath if there's a section path